src/loaddb/load_common.cpp

```diff
@@ -151,7 +151,7 @@ namespace cubload
   }
 
   size_t
-  batch::get_packed_size (cubpacking::packer &serializator) const
+  batch::get_packed_size (cubpacking::packer &serializator, std::size_t start_offset) const
```

Comment:
please open a different patch for packing changes :)

Reply:
I fixed packing issue on base branch and merged it with current one, should be fixed